Cargo.toml

```diff
@@ -44,9 +44,12 @@ serde = ["dep:serde", "bitflags/serde"]
 ## Enables raw file descriptor polling / selecting instead of mio.
 use-dev-tty = ["filedescriptor", "rustix/process"]
 
+## Enables `is_*` and `as_*` helpers for event enums.
+derive-more = ["dep:derive_more"]
```

Comment:
small nit: should this be `derive_more` to match the crate name, or `derive-more` to align with the feature flag conventions used here?

Reply:
I think it's better to match the existing feature flags here but I don't have a strong preference

Reply:
Yeah, I'm 70/30 on that (i.e. I agree with you, but wanted to check that that this was correct because I wasn't sure).